sql/core/src/main/scala/org/apache/spark/sql/execution/CacheManager.scala

```diff
@@ -79,10 +91,11 @@ class CacheManager extends Logging with AdaptiveSparkPlanHelper {
     if (lookupCachedData(planToCache).nonEmpty) {
       logWarning("Asked to cache already cached data.")
     } else {
-      // Turn off AQE so that the outputPartitioning of the underlying plan can be leveraged.
-      val sessionWithAqeOff = getOrCloneSessionWithAqeOff(query.sparkSession)
-      val inMemoryRelation = sessionWithAqeOff.withActive {
-        val qe = sessionWithAqeOff.sessionState.executePlan(planToCache)
+      // Turn off configs so that the outputPartitioning of the underlying plan can be leveraged.
+      val sessionWithConfigsOff = getOrCloneSessionWithConfigsOff(
```

Comment:
nit:  it seems we don't this line break;
```
      val sessionWithConfigsOff = getOrCloneSessionWithConfigsOff(query.sparkSession, configsOff)
```

Reply:
@maropu - updated.